src/connect.js

```diff
@@ -221,4 +224,16 @@ export function mapParams(paramKeys, params) {
 	return result;
 }
 
+const handleAsyncOnDatas = async response => {
+	try {
+		response = await response;
+		if (response.then) {
+			response = await handleAsyncOnDatas(response);
+		}
+		return response;
+	} catch (ex) {
+		throw ex;
```

Comment:
why `catch`, do nothing and then `throw` ? This is equivalent to not having the `catch` block at all.